### PR TITLE
Text of all minutes files get written to database

### PR DESCRIPTION
**Pull request by GuyLiner**
- Config file to determine location of database and minutes folder
- PDF to database
- Multithreading for downloading files
- Multiproc for tesseract
#7

### CODE REVIEW

File: src/cobb_tracker/cobb_config.py [hunk 1]
```diff
@@ -0,0 +1,50 @@
+import configparser
+import sys
+import os
+from pathlib import Path
+
+class cobb_config():
```
Comment (tyler-8) [inline src/cobb_tracker/cobb_config.py]: Let's use some PEP8-like style. Classes are `CapWords`, functions & variables are `snake_case`

```python
class CobbConfig:
```

File: src/cobb_tracker/cobb_config.py [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+    def __init__(
+            self,
+            config_dir = os.path.join(Path(os.path.expanduser('~')),
```
Comment (tyler-8) [inline src/cobb_tracker/cobb_config.py]: If we're using `pathlib` we don't need `os.path`.

```python
p = pathlib.Path('parent_folder')
p.exists()
child_folder = p.joinpath("child_folder")


```

File: src/cobb_tracker/cobb_config.py [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+                                      ".local",
+                                      "share")
+            ):
+        self.DEFAULT_CONFIG_DIR=config_dir
+        self.DEFAULT_DATA_DIR=data_dir
+        self.DEFAULT_CONFIG_FILE = os.path.join(f"{self.DEFAULT_CONFIG_DIR}","config.ini")
+        self.config = configparser.ConfigParser()
+
+        if not os.path.exists(self.DEFAULT_CONFIG_DIR):
+            os.mkdir(self.DEFAULT_CONFIG_DIR)
```
Comment (tyler-8) [inline src/cobb_tracker/cobb_config.py]:
```python
self.DEFAULT_CONFIG_DIR.mkdir(exists_ok=True)
```

File: src/cobb_tracker/municipalities/file_ops.py [hunk 4]
```diff
@@ -1,32 +1,85 @@
-import pathlib
+from pathlib import Path
 import sys
 import os
 import requests
+import concurrent.futures
```
Comment (tyler-8) [inline src/cobb_tracker/municipalities/file_ops.py]: You can shorten later usage by doing

```python
import concurrent.futures as cf

cf.ThreadPoolExecutor()
```

**Review by tyler-8 — CHANGES_REQUESTED** [review]
Looking good. I think we should probably consider adding the `ruff` formatter/linter to the project. It'll help with some of the formatting and keeping it consistent.

**Comment by GuyLiner** [comment]
> Looking good. I think we should probably consider adding the `ruff` formatter/linter to the project. It'll help with some of the formatting and keeping it consistent.

Yeah, I definitely agree

**Review by tyler-8 — APPROVED** [review]
LGTM!

**Pull request merged**